<launch>

  <node name="speech_interface_node"
        pkg="allergen_scanner"
        type="speech_interface_node.py"
        output="screen" />

  <node name="vision_node"
        pkg="allergen_scanner"
        type="vision_node.py"
        output="screen" />

  <node name="allergen_manager_node"
        pkg="allergen_scanner"
        type="allergen_manager_node.py"
        output="screen"
        launch-prefix="bash -c 'sleep 3; $0 $@' "/>

</launch>